src/reducers/signinReducer.js

```diff
@@ -0,0 +1,28 @@
+import {SIGNIN_FETCH, SIGNIN_SUCCESS, SIGNIN_FAILURE} from '../actions/signInActions'
+
+const initialState ={
+    user_id: 0,
```

Comment:
We should want to include their email, first_name, last_name etc... include the entire user entity.